tests/database/mysql/mysqlTable.test.js

```diff
@@ -166,26 +166,72 @@ describe("database/mysql/mysqlTable", () => {
       [
         {
           name: "it-3",
-          order: 2
+          order: 2,
         },
         {
           name: "it-2",
-          order: 1
+          order: 1,
         },
         {
           name: "it-1",
-          order: 3
-        }
-      ].forEach(async item => {
+          order: 3,
+        },
+      ].forEach(async (item) => {
         await table1.setItem(null, {
           id: item.name,
           name: item.name,
-          order: item.order
+          order: item.order,
         });
       });
 
       const items = await table1.getItems();
       expect(items.map((i) => i.id)).toEqual(["it-2", "it-3", "it-1"]);
     });
   });
+
+  describe("collection size", () => {
+    it("size without query", async () => {
+      const database = dbCreate({ descriptor, ...dbConfig });
+      await database.reset();
+
+      const table1 = database.getTable("table1");
+
+      let count = await table1.size();
+      expect(count).toEqual(0);
+
+      await table1.setItem(null, {
+        id: "id1",
+      });
+      count = await table1.size();
+      expect(count).toEqual(1);
+
+      await table1.setItem(null, {
+        id: "id2",
+      });
+      count = await table1.size();
+      expect(count).toEqual(2);
+    });
+    it("size with query", async () => {
```

Comment:
add a blank line before the next `it` block

Reply:
and add a better explanation for this test case too

Reply:
is this ok ?